packages/connect-ui/src/components/Notification.tsx

```diff
@@ -0,0 +1,159 @@
+`;
+
+const NotificationBody = styled.div`
+    font-weight: 300;
+    font-size: 14px;
+    color: #eb8a00;
+`;
+
+const NotificationCta = styled.div`
+    padding-top: 8px;
+`;
+
+const StyledButton = styled(Button)`
+    width: 100%;
+    border: 1px solid #eb8a00;
+    background-color: transparent;
+    text-align: center;
+    color: #eb8a00;
+    transition: all 0.15s;
+
+    &:hover {
+        background-color: #eb8a00;
+        color: #ffefd9;
+    }
+
+    &:active {
+        background-color: #eb8a00;
+        color: #ffefd9;
+    }
+
+    &:focus {
+        background-color: #eb8a00;
+        color: #ffefd9;
+    }
+`;
+
+interface NotificationProps {
+    header: string;
+    body: string;
+    variant: 'warning' | 'danger';
+    cta: {
+        desc: string;
+        url: string;
+    };
+}
+
+const Notification = ({ header, body, cta, variant }: NotificationProps) => {
+    const [hidden, setHidden] = useState(false);
+
+    if (hidden) {
+        return null;
+    }
+
+    // todo: find some nice red colors
+    const color = variant === 'warning' ? '#ffefd9' : '#ffefd9';
+
+    return (
+        <NotificationBox color={color}>
+            <NotificationLeftCol>
+                <Icon icon="INFO" color="#eb8a00" size={20} />
+            </NotificationLeftCol>
+            <NotificationRightCol>
+                <NotificationHeader>
+                    {header}
+                    <Icon icon="CROSS" color="#eb8a00" onClick={() => setHidden(true)} />
+                </NotificationHeader>
+                <NotificationBody>
+                    <div>{body}</div>
+                </NotificationBody>
+                <NotificationCta>
+                    <StyledButton
+                        onClick={() => {
+                            window.open(cta.url);
+                            window.close();
+                        }}
+                    >
+                        {cta.desc}
+                    </StyledButton>
+                </NotificationCta>
+            </NotificationRightCol>
+        </NotificationBox>
+    );
+};
+
+export const FirmwareUpdateNotification = () => (
+    <Notification
+        variant="warning"
+        header="New firmware update available"
+        body="A new firmware update is available for your Trezor device. Update now
+for the latest features."
+        cta={{ desc: 'Update my firmware', url: SUITE_FIRMWARE_URL }}
+    />
+);
+
+export const BackupNotification = () => (
+    <Notification
+        variant="warning"
+        header="Your Trezor is not backed up"
+        body=""
+        cta={{
+            desc: 'Create a backup in 3 minutes',
+            url: SUITE_BACKUP_URL,
+        }}
+    />
+);
+
+export const BridgeUpdateNotification = () => (
+    <Notification
+        variant="warning"
+        header="New bridge is available"
+        body=""
+        cta={{ desc: 'Update bridge', url: SUITE_BRIDGE_URL }}
+    />
+);
+
+export const SuspiciousOriginNotification = () => (
```

Comment:
Perhaps put those notifications into a separate file?

Reply:
personally, I like it more here. It is not comlex enough to have own file imho, so I would stick to this unless we have some code-guidelines/linting rules that enforce it.